### PR TITLE
Language Support PT-BR

### PR DESCRIPTION
**Pull request by LeandroStanger**

### CODE REVIEW

File: po/pt_BR.po [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+msgid "Your SteamID will be read from Steam configuration file"
+msgstr "Seu SteamID será lido a partir do arquivo de configuração do Steam"
+
+#: src/data/sources/steam/Steam.vala:20
+msgid ""
+"Steam config file not found.\n"
+"Login into your account in Steam client and return to GameHub"
+msgstr ""
+"Steam config file not found.\n"
+"Entre na sua conta no cliente Steam e retorne ao GameHub"
```
Comment (tkashkin) [inline po/pt_BR.po]: @LeandroStanger Please translate this line.

File: po/pt_BR.po [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+#: src/ui/views/GamesView/GamesView.vala:142
+msgid "Search"
+msgstr "Pesquisar"
+
+#: src/ui/views/GamesView/GamesView.vala:183
+#, c-format
+msgid "%u game"
+msgid_plural "%u games"
+msgstr[0] ""
+msgstr[1] ""
```
Comment (tkashkin) [inline po/pt_BR.po]: @LeandroStanger And this plural (should be "one game" and "many games" if pt_BR has two plural forms)

File: po/pt_BR.po (hunk 1 above)
Comment (tkashkin) [reply po/pt_BR.po]: @LeandroStanger it's multiline string and should be translated like [this](https://github.com/tkashkin/GameHub/blob/master/po/ru.po#L66-L72). Syntax:
```
msgstr ""
msgstr "line1\n"
msgstr "line2"
```
All lines except last should end with \n

File: po/pt_BR.po [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+msgid "Your SteamID will be read from Steam configuration file"
+msgstr "Seu SteamID será lido a partir do arquivo de configuração do Steam"
+
+#: src/data/sources/steam/Steam.vala:20
+msgid ""
+"Steam config file not found.\n"
+"Login into your account in Steam client and return to GameHub"
+msgstr "Entre na sua conta no cliente Steam e retorne ao GameHub"
+"Steam config file not found.\n"
+"Entre na sua conta no cliente Steam e retorne ao GameHub"
```
Comment (tkashkin) [inline po/pt_BR.po]: @LeandroStanger it's multiline string and should be translated like [this](https://github.com/tkashkin/GameHub/blob/master/po/ru.po#L66-L72). Syntax:
```
msgstr ""
msgstr "line1\n"
msgstr "line2"
```
All lines except last should end with \n

File: po/pt_BR.po [hunk 4]
```diff
@@ -66,8 +66,8 @@ msgstr "Seu SteamID será lido a partir do arquivo de configuração do Steam"
 msgid ""
 "Steam config file not found.\n"
 "Login into your account in Steam client and return to GameHub"
-msgstr "Entre na sua conta no cliente Steam e retorne ao GameHub\n"
-"Steam config file not found.\n"
+msgstr ""
```
[the remark this answers is not in the document]
Comment (LeandroStanger) [reply po/pt_BR.po]: Correct formatting

**Comment by tkashkin** [comment]
@LeandroStanger Thanks for your contribution. It will be included in the next release.

**Comment by LeandroStanger** [comment]
@cho2 Could you translate for Indonesian?

**Comment by cho2** [comment]
@LeandroStanger let me try

**Pull request merged**